### ISSUE TITLE
always allow you to see/delete your own community posts, even if no longer a member of the comm

### ISSUE THREAD

**Issue by rahaeli** [dreamwidth/dw-free#1329]
Pretty sure we had a bugzilla bug for this, but I can't remember the details...

Anyway, even once you've left or been removed from a community, you should still be able to see *your own* members-locked entries, even if you're not a member of the community anymore. (This will allow you to delete them if you want and prevent you from being locked out of your own content, etc.) 

I think we can probably handle this just by checking "are you a member of the community OR the poster of the entry" for "should you see a members-locked post in a community", but there should probably be a bunch of testing...

**Comment by kareila** [comment]
Giving a user full access to their own entries in any community doesn't look too hard - it might be as simple as updating the `visible_to` method in LJ::Entry. But if we want to give those users only read/delete access and disallow them from editing or further commenting on their entries, I think that will require mucking about with LJ::Protocol.

**Comment by rahaeli** [comment]
Hm. I don't see a problem with allowing them to edit the entry. Allowing them to comment in the (members-locked) entry without being a member of the comm would be a problem, but wouldn't that be fixable by just adding a "if you're a member" check to the "can you comment to this entry" check?

**Comment by kareila** [comment]
Yeah, it looks like the `does_not_allow_comments_from` user method might work for that in the web interface, but I still think it would need to be fixed for `addcomment` in LJ::Protocol as well.

**Comment by kaberett** [comment]
https://dw-suggestions.dreamwidth.org/417928.html

**Comment by kaberett** [comment]
Another request: https://www.dreamwidth.org/support/see_request?id=36993

**Comment by rahaeli** [comment]
Had another request about this today, so bumping it to see if anyone wants to take it on, etc.
